database/model_config.go

```diff
@@ -19,6 +19,6 @@ type SQLiteConfig struct {
 var InMemorySqliteConfig = DatabaseConfig{
 	DatabaseName: "sqlite",
 	SQLite: &SQLiteConfig{
-		Path: ":memory:",
+		Path: "file::memory:?cache=shared",
```

Comment:
`file::` looks wrong - is that correct?

Reply:
We need the `file::` to make it a public cache. Ref: https://www.sqlite.org/inmemorydb.html